Lib/sqlite3/test/hooks.py

```diff
@@ -260,6 +261,9 @@ def trace(statement):
         con2.execute("create table bar(x)")
         cur.execute(queries[1])
         self.assertEqual(traced_statements, queries)
+        con1.close()
+        con2.close()
+        gc_collect()
```

Comment:
Could you add a small comment why we need to explicitly call `gc_collect()`?

Reply:
Yes, I actually had that in mind.